introduction.md

```diff
@@ -1 +1 @@
-I love Ruby and JavaScript!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?